src/pytesmo/timedate/dekad.py

```diff
@@ -1,314 +1,8 @@
-    return startdate
-
-
-def check_dekad_enddate(dt):
-    """
-    Check if a date is a dekad enddate
-    """
-    return check_dekad(dt) == dt
-
-
-def check_dekad_startdate(dt):
-    """
-    Check if a date is a dekad startdate
-    """
-    if dt.day in [1, 11, 21]:
-        return True
-    else:
-        return False
-
-
-def group_into_dekads(dates, use_dekad_startdate=False):
-    """
-    Group a list of dates into dekads.
-
-    Parameters
-    ----------
-    dates: list of datetime.datetime
-    use_dekad_startdates: boolean, optional
-        If set the dekad reference dates will
-        be the startdates of the dekad
-
-    Returns
-    -------
-    groups: dict
-        keys: dekad reference dates
-        values: list of dates belonging to dekad
-    """
-    groups = {}
-    for dt in dates:
-        dekad_date = check_dekad(dt)
-        if use_dekad_startdate:
-            dekad_date = dekad_startdate_from_date(dekad_date)
-        if dekad_date not in groups:
-            groups[dekad_date] = []
-        groups[dekad_date].append(dt)
-    return groups
-
-
-def dekad2day(year, month, dekad):
-    """Gets the day of a dekad.
-
-    Parameters
-    ----------
-    year : int
-        Year of the date.
-    month : int
-        Month of the date.
-    dekad : int
-        Dekad of the date.
-
-    Returns
-    -------
-    day : int
-        Day value for the dekad.
-    """
-
-    if dekad == 1:
-        day = 10
-    elif dekad == 2:
-        day = 20
-    elif dekad == 3:
-        day = calendar.monthrange(year, month)[1]
-
-    return day
-
-
-def runningdekad2date(year, rdekad):
-    """Gets the date of the running dekad of a spacifc year.
-
-    Parameters
-    ----------
-    year : int
-        Year of the date.
-    rdekad : int
-        Running dekad of the date.
-
-    Returns
-    -------
-    datetime.datetime
-        Date value for the running dekad.
-    """
-
-    month = int(math.ceil(rdekad / 3.))
-    dekad = rdekad - month * 3 + 3
-    day = dekad2day(year, month, dekad)
-
-    return datetime(year, month, day)
-
-
-def day2dekad(day):
-    """Returns the dekad of a day.
-
-    Parameters
-    ----------
-    day : int
-        Day of the date.
-
-    Returns
-    -------
-    dekad : int
-        Number of the dekad in a month.
-    """
-
-    if day < 11:
-        dekad = 1
-    elif day > 10 and day < 21:
-        dekad = 2
-    else:
-        dekad = 3
-
-    return dekad
-
-
-def get_dekad_period(dates):
-    """Checks number of the dekad in the current year for dates given as list.
-
-    Parameters
-    ----------
-    dates : list of datetime
-        Dates to check.
-
-    Returns
-    -------
-    period : list of int
-        List of dekad periods.
-    """
-
-    period = []
-
-    for dat in dates:
-
-        d = check_dekad(dat)
-        dekad = day2dekad(d.day)
-        per = dekad + ((d.month - 1) * 3)
-        period.append(per)
-
-    return period
+from cadati.dekad import *
```

Comment:
Dekad function are now in cadata, but the still can be imported from pytesmo, for now